release-notes.md

```diff
@@ -1,3 +1,54 @@
+# Quasar-17.17.0
+## Changes EmailEventNotification co-editors or co-publishers changed templates ##
```

Comment:
change to
 Changes to co-editors or co-publishers changed email event notification templates